packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -4594,6 +4630,12 @@ msgstr "článků blogu"
 msgid "brands"
 msgstr "značek"
 
+msgid "column {{ column_name }} cannot be empty"
```

Comment:
It is just a nitpick, however, should not these messages be rather in `validators` domain? I believe `Translator::VALIDATOR_TRANSLATION_DOMAIN` should be used in `PriceListFacade::importPriceList()`

Reply:
ok, eebb60290d